src/app/app.component.ts

```diff
@@ -68,9 +68,9 @@ export class AppComponent {
     roles['ROLE_DPO'] = roles['ROLE_CONTROLLER'].concat([
       'CanCreatePIA', 'CanCreatePIAExample', 'CanShowPIA',
       'CanEvaluatePIA', 'CanValidatePIA', 'CanCancelValidatePIA',
-      'CanDeletePIA', 'CanCreateFolder',
-      'CanCreateProcessing', 'CanShowProcessing', 'CanDeleteProcessing',
-      'AccessToValidationSection'
+      'CanDeletePIA', 'CanImportPIA', 'CanExportPIA', 'CanCreateFolder',
```

Comment:
The role `DPO` don't have rights to import / export PIA. Please move it under the role `SUPER_ADMIN`.